README.md

```diff
@@ -140,11 +142,16 @@ permissions have been set to allow it.
 }
 ```
 
-You can then run `chalice deploy --no-autogen-policy`.
+You can then run `chalice deploy --staging commonsstaging --no-autogen-policy`.
```

Comment:
It seems like we should be able to roll out a couple of different environments. Is there a reason we are specifying commonsstaging in the readme?